src/main/webapp/i18n/de/editor.json

```diff
@@ -17,7 +17,7 @@
             "unsubmittedChanges": "Nicht abgesendete Änderungen vorhanden.",
             "submittingChanges": "Änderungen werden abgesendet.",
             "preparingForSubmit": "Absendung wird vorbereitet.",
-            "submit": "Absenden",
+            "submit": "Kompilieren und Ausführen",
```

Comment:
please revert this change!

Reply:
Sorry for that. Somehow it slipped in to my branch. Reverted the changed. What do you mean by tooltip?